ci/cases/pr/C96_atmsnowDA.yaml

```diff
@@ -0,0 +1,20 @@
+experiment:
+  system: gfs
+  mode: cycled
+
+arguments:
+  pslot: {{ 'pslot' | getenv }}
+  app: ATM
+  resdet: 96
+  comrot: {{ 'RUNTESTS' | getenv }}/COMROT
+  expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
+  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C96C48
+  idate: 2021122012
+  edate: 2021122100
+  nens: 0
+  gfs_cyc: 0
```

Comment:
I think @jiaruidong2017 had @KateFriedman-NOAA add in gfs dumps for the test. Up to you to test before merging or in a later PR when we do IAU we can add in gfs_cyc 1

Reply:
Yes.  @KateFriedman-NOAA has added the obs for `gfs_cyc=1`.